<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NcSessionWidget</class>
 <widget class="QWidget" name="NcSessionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="verticalWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="headerGroupBox">
        <property name="title">
         <string>Session</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="hostnameLabel">
           <property name="text">
            <string>Hostname</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="addressComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Hostname in IP or Domain Name</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtTop</enum>
           </property>
           <property name="minimumContentsLength">
            <number>0</number>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="portComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Range between 0 and 65535</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtTop</enum>
           </property>
           <item>
            <property name="text">
             <string>80</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8080</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>73</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CustomPlainTextEdit" name="sessionPlainTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="footerLayoutWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="CustomPlainTextEdit" name="userInputPlainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Use Shift+Enter for new line</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="toolTip">
         <string>Send Message</string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>src/widgets/customplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
